webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/EventListDisplayer.scala

```diff
@@ -80,6 +80,7 @@ import com.normation.rudder.rule.category.RoRuleCategoryRepository
 import org.joda.time.format.DateTimeFormat
 import com.normation.rudder.web.model.LinkUtil
 import org.joda.time.format.ISODateTimeFormat
+import scala.concurrent.Future
 import com.normation.box._
 
```

Comment:
I think you need to delete the code that is not used anymore to display a events here